### DISCUSSION TITLE
Restrict levels in "contents" sidebar

### DISCUSSION THREAD

**Discussion by michael-rapp** [pradyunsg/furo#728] [Ideas] [answered]
I was asking myself whether it is possible to restrict the number of levels shown in the "contents" sidebar (on the right). In my particular use case, I want to restrict the sidebar to only include top-level headings (those with the tag `<h1>`, I guess). If this is currently not possible, I think it would be a useful enhancement.

**Comment by pradyunsg** [comment, marked as answer]
https://github.com/pradyunsg/furo/discussions/566#discussioncomment-4422331 describes this!

**Reply by michael-rapp** [reply]
Great! This is exactly what I was looking for. Thank you very much!
